test/dummy/app/views/layouts/application.html.erb

```diff
@@ -6,7 +6,8 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application" %>
+    <%# Includes all stylesheet files in app/assets/stylesheets %>
+    <%= stylesheet_link_tag :app, "data-turbo-track": "reload" %>
```

Comment:
This is what Rails 8 generates. Should this PR remove `"data-turbo-track": "reload"` as well?